packages/app/design-system/text-input.tsx

```diff
@@ -1,17 +1,18 @@
 import { ComponentProps } from 'react'
 import { TextInput as DripsyTextInput } from 'dripsy'
+import { Platform } from 'react-native'
 
 import { tw as tailwind } from 'app/design-system/tailwind'
 
 type TextProps = {
-  tw?: string
+  className?: string
 } & ComponentProps<typeof DripsyTextInput>
 
-function TextInput({ tw, sx, ...props }: TextProps) {
+function TextInput({ className, sx, ...props }: TextProps) {
   return (
     <DripsyTextInput
-      sx={{ ...sx, ...tailwind.style(tw) }}
-      style={{ outlineStyle: 'none' } as any}
+      sx={{ ...sx, ...tailwind.style(className) }}
+      style={Platform.OS === 'web' ? ({ outlineStyle: 'none' } as any) : {}}
```

Comment:
not sure if your rn tailwind implementation supports platform prefixes? 
If yes, it might make sense to move to one that does, so you could do `web:outline-none`?

Reply:
it should work but isn't, will look into it later down the line

Reply:
you probably don't have tw context working properly?